eth/main.cpp

```diff
@@ -382,13 +382,13 @@ int main(int argc, char** argv)
 			return -1;
 		}
 
-#if ETH_EVMJIT
+#if defined(ETH_EVMJIT) || defined(ETH_HERA)
 	if (vm.count("vm"))
 	{
 		string vmKind = vm["vm"].as<string>();
 		if (vmKind == "interpreter")
 			VMFactory::setKind(VMKind::Interpreter);
-		else if (vmKind == "jit")
+		else if (vmKind == "jit" || vmKind == "hera")
```

Comment:
Because the JitVM is not modified for Hera, it will not work at the moment, right?

Reply:
Updated.